cmd/buildah/manifest.go

```diff
@@ -659,12 +657,15 @@ func manifestPushCmd(c *cobra.Command, args []string, opts manifestPushOpts) err
 	if err != nil {
 		return err
 	}
-
 	systemContext, err := parse.SystemContextFromOptions(c)
 	if err != nil {
 		return errors.Wrapf(err, "error building system context")
 	}
 
+	return manifestsPush(systemContext, store, listImageSpec, destSpec, opts)
+}
+
+func manifestsPush(systemContext *types.SystemContext, store storage.Store, listImageSpec, destSpec string, opts pushOptions) error {
```

Comment:
Rename to `manifestPush`.  The stray "s" isn't consistent with anything.

Reply:
Strange I had it their originally and then mistakenly made this change. C hanged back.